bears/general/SpaceConsistencyBear.py

```diff
@@ -38,6 +56,9 @@ def run(self,
         result_texts = []
         additional_info_texts = []
 
+        if self.section.aspects and Indentation in self.section.aspects:
+            use_spaces = Indentation.indent_type == 'space'
```

Comment:
you made `use_spaces` to be a bool right?

Reply:
yes, what's the issue

Reply:
so you are assigning it a string

Reply:
no it's True if Indentation.indent_type == 'space' and false otherwise

Reply:
but perhaps writing this line as 
```use_spaces = (Indentation.indent_type == 'space')```
would make things clearer

Reply:
ohh I didnt see the `==` before 'space' my bad ;P